src/main/java/org/scijava/io/BytesHandle.java

```diff
@@ -231,26 +231,31 @@ public short readShort() throws IOException {
 	public void write(final byte[] b, final int off, final int len)
 		throws IOException
 	{
-		validateLength(len);
+		ensureWritable(len);
 		bytes().put(b, off, len);
 	}
 
 	@Override
 	public void write(final int b) throws IOException {
-		validateLength(1);
+		writeByte(b);
```

Comment:
Confusing that these changes to `BytesHandle` happen in a commit purportedly about `DataHandle` default methods.